common/plugin_control.c

```diff
@@ -539,26 +541,21 @@ static char **csv_split(const char *s, size_t *field_count)
 
         size_t i = 0;
         result[i] = copy;
-        size_t qcount = 0;
         for (len = 0; s[len]; ++len) {
-            if (s[len] == ',' && !in_string) {
-                copy++;
-                result[++i] = copy;
-                qcount = 0;
-            } else if (s[len] == '"' && !in_string && (len == 0 || s[len - 1] == ',')) {
-                /* Leading quote */
-                in_string = true;
-            } else if (s[len] == '"' && in_string && (s[len + 1] == ',' || s[len + 1] == '\0')) {
-                /* Trailing quote */
-                in_string = false;
-            } else if (s[len] == '"' && in_string) {
-                /* ignore the first double quote we see in a string */
-                if (++qcount % 2 == 0) {
+            if (s[len] == '\\') {
+                ++len;
```

Comment:
Once again, if the string ended with a single `\`, then I think we'd run off the end of the string.